app/src/main/java/com/zeapo/pwdstore/ui/onboarding/fragments/RepoLocationFragment.kt

```diff
@@ -66,6 +74,26 @@ class RepoLocationFragment : Fragment(R.layout.fragment_repo_location) {
         repositoryInitAction.launch(UserPreference.createDirectorySelectionIntent(requireContext()))
     }
 
+    private val gpgKeySelectAction = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) { result ->
+        if (result.resultCode == AppCompatActivity.RESULT_OK) {
```

Comment:
This is a reasonable solution. Alternatively, we could run this flow anytime we encounter an empty `.gpg-id`, which could help more users get out of this situation.

Reply:
hm I can make it happen, one sec.

Reply:
This works now